test/runtime.jl

```diff
@@ -8,13 +8,12 @@
     @testset "custom logs_dir" begin
         mktempdir() do logs_dir
             cp("/tmp/ray/session_latest/logs/raylet.out", joinpath(logs_dir, "raylet.out"))
+            err = IOBuffer()
             code = quote
                 using Ray
                 Ray.init(; logs_dir=$logs_dir)
             end
-            cmd = `$(Base.julia_cmd()) --project=$(Ray.project_dir()) -e $code`
-            err = IOBuffer()
-            run(pipeline(cmd; stderr=err))
+            process_eval(code; stderr=err)
```

Comment:
Avoiding using the macro here as otherwise `$log_dir` is attempted to be interpreted on the spawned Julia process. Doing things this way cause the `$logs_dir` to be expanded on the parent Julia process.